src/api/app/models/bs_request_action.rb

```diff
@@ -757,9 +757,6 @@ def check_action_permission_source!
 
   def check_action_permission!(skip_source = nil)
     # find objects if specified or report error
-    role=nil
```

Comment:
This can't be removed because is used in line 786 and if `self.role` is false then the local variable `role` is undefined there